airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -16,10 +16,12 @@
 # under the License.
 
 from airflow.exceptions import AirflowException
+from airflow.kubernetes.kube_client import get_kube_client
+from airflow.kubernetes.pod import Resources
+from airflow.kubernetes.pod_generator import PodGenerator
+from airflow.kubernetes.pod_launcher import PodLauncher
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
-from airflow.kubernetes import kube_client, pod_generator, pod_launcher
-from airflow.kubernetes.pod import Resources
```

Comment:
Do we need to do changes like this? What's the problem with the existing import?

Reply:
I think that was a try from @mik-laj to get rid of the kubernetes import problem by autoapi 1.1.0 (which is quite illogical). Once we see that pinning autoapi works, we should likely go back with those changes

Reply:
Probably not necessary. Kubernetes was the only problem that I was unable to solve in the first commit and tried to solve the problem by trial and error.
Reference: https://issues.apache.org/jira/browse/AIRFLOW-4841?focusedCommentId=16870850&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16870850

Reply:
Can you make a fixup and revert those @mik-laj - before we merge the change with autoapi pinned ?

Reply:
yes.  I will do it, but I am currently busy.